src/main/java/dk/bec/polonez/reservationsystem/dto/notificationDto/ReservationStatusNotificationInput.java

```diff
@@ -0,0 +1,13 @@
+package dk.bec.polonez.reservationsystem.dto.notificationDto;
+
+import lombok.Data;
+
+@Data
+public class ReservationStatusNotificationInput {
+
+    private String username;
+
+    private String reservationInfo;
+
+    private String reservationStatus;
```

Comment:
**advice** What about using enumeration (```enum```) for status with finite number of possible values? It would make the code more explicit and readable.

Reply:
We can create an enum. But in case of this particular DTO, shouldn't it store status as a string? (we can call toString() on enum object when converting to DTO object). Because it gets inserted directly into email content.

Reply:
Yes, you can should ```enum``` as much as possible in your code and then you should serialize your ```enum``` to String (by calling ```.toString()```) only if you really need it and if it's unavoidable - for example just before you send an email when you are preparing message:

```
String message = String.format(
                template.getMessage(),
                input.getUsername(),
                input.getReservationInfo(),
                input.getReservationStatus().toString()); // enum.toString()
```

But luckily ```String.format()``` that you're using should deal with ```enum``` without a need to convert to ```String```:

```
String message = String.format(
                template.getMessage(),
                input.getUsername(),
                input.getReservationInfo(),
                input.getReservationStatus()); // enum without converting to String (it should work)
```